tests/integration/components/frost-bunsen-select-test.js

```diff
@@ -0,0 +1,83 @@
+import {expect} from 'chai'
+import {describeComponent, it} from 'ember-mocha'
+import {beforeEach, describe, mocha} from 'mocha'
+import {integrationTestContext, renderWithProps} from 'dummy/tests/helpers/template'
+
+mocha.setup({
+  timeout: 10000
+})
+
+const cellConfig = {
+  model: 'name',
+  renderer: 'select',
+  writeTransforms: [
+    {
+      object: {
+        id: '${id}',
+        index: '${index}',
+        label: '${label}',
+        literal: 'foo',
+        value: '${value}'
+      }
+    }
+  ]
+}
+
+const view = {
+  containers: [
+    {
+      id: 'main',
+      rows: [
+        [cellConfig]
+      ]
+    }
+  ],
+  rootContainers: [{
+    container: 'main',
+    label: 'Main'
+  }],
+  type: 'form',
+  version: '1.0'
+}
+
+describeComponent(...integrationTestContext('frost-bunsen-input-select'), function () {
```

Comment:
We've taken it even further in a few of the test files https://github.com/ciena-frost/ember-frost-bunsen/blob/master/tests/integration/components/frost-bunsen-container-test.js#L18